oeps/architectural-decisions/oep-0041-arch-async-server-event-messaging.rst

```diff
@@ -433,8 +437,8 @@ outages over the years, particularly when course-discovery becomes overloaded
 with edx-platform generated requests.
 
 
-Eliminate Expensive, Batch Synchronization
-------------------------------------------
+Eliminate Expensive, Delayed, Batch Synchronization
```

Comment:
Hopefully some of these changes will land in a separate PR: https://github.com/openedx/open-edx-proposals/pull/321/files